healthScore/views.py

```diff
@@ -284,7 +212,7 @@ def registration(request):
             )
             return render(request, "registration.html", context)
 
-        elif user.objects.filter(userName=username).exists():
+        elif user.objects.filter(username=username).exists():
```

Comment:
Won't these have to be userName? Since filter will search for the userName key in the db?

Reply:
Sarthak adjusted the DB column header to be "username" to deal with an issue I was running into

Reply:
Ohh, alright